discord/monetization.py

```diff
@@ -105,6 +113,8 @@ class Entitlement:
     user_id: Optional[:class:`int`]
         :attr:`~discord.SKUFlags.app_user_subscription` only: The ID of the user that is granted access to the
         entitlement's sku
+    type: :class:`discord.EntitlementType`
```

Comment:
Properties should only be documented in the property (getter) function itself, see suggestion bellow.

Reply:
Please remove this as it is documented in the property